web/cobrands/sass/_base.scss

```diff
@@ -1326,17 +1326,25 @@ input.final-submit {
   }
 }
 
-.item-list--wards__item {
-  @extend %item-list__item--link;
+.item-list--wards {
+  @extend %item-list;
 
-  label {
-    margin: 0;
-    padding: 0.5em 1em;
+  &.govuk-checkboxes--small {
+    .govuk-checkboxes__item {
+      float: none; // Overrides float:left from govuk-checkboxes--small
+      margin-left: 1em; // As otherwise no left margin
+    }
+  }
+
+  a {
+    display: block;
+    margin-left: calc(-1em + -34px); // for the padding-left on govuk-checkboxes__item + padding of item-list--wards
+    margin-bottom: 0.5rem;
   }
 }
 
 .item-list--reports__item {
-  @extend %item-list__item--link;
+  @extend %item-list;
```

Comment:
We've still got the issue here with this here is a child `<li>` `.item-list--reports__item` extending `%item-list`, and above we have the `.item-list--wards` `<ul>` extending `%item-list`, which feels wrong.